pkgs/top-level/all-packages.nix

```
@@ -21695,6 +21695,8 @@ in
 
   jormungandr = callPackage ../applications/blockchains/jormungandr { };
 
+  ledgeragent = callPackage ../applications/misc/ledgeragent { };
```

```suggestion
   ledgeragent = with python3Packages; toPythonApplication ledgeragent;
```

This preferred over adding extra files in the nixpkgs tree.
See other examples in `all-packages.nix`

---

Thanks, good call. Done.